src/seedu/addressbook/data/person/Address.java

```diff
@@ -56,4 +77,19 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+	   public String getAddressBlock() {
+    	return addressBlock.getValue();
+    }
+    
+    public String getAddressUnit() {
```

Comment:
how about bundling all of these into the method toSting() and returning a complete whole address to the user?